### DISCUSSION TITLE
Welcome to obsidian-to-ical-plugin Discussions!

### DISCUSSION THREAD

**Discussion by andrewbrereton** [andrewbrereton/obsidian-to-ical-plugin#65] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nilvanlopes** [comment]
Hi
I am from Brazil
Is there any way to work with the day planner where the title is in the format DD-MM-YY
Really like your work
![image](https://github.com/andrewbrereton/obsidian-to-ical-plugin/assets/28903634/b52f2899-7d40-4fab-9c00-791b05c2ca79)

**Reply by andrewbrereton** [reply]
Hi @nilvanlopes, this is a good suggestion. I will add support for a customisable date format in a future version.

**Reply by Vulcaas** [reply]
Hi, is this feature implemented yet?
